.github/ISSUE_TEMPLATE/docs.md

```diff
@@ -0,0 +1,31 @@
+---
+name: 'Drift docs: Issue / Feature request'
+about: Create a report to help improve drift's docs
+title: ''
+labels: ['docs']
+assignees: ''
+---
+
+**Description**
+
+Thanks for taking the time to file an issue for drift's docs!
+
+For feature requests, please describe the feature you're interested in.
+
+For information requests, please describe what you think is missing from
+the docs and if possible where should this information be placed.
+
+For bug reports, please describe what is wrongand what do you believe should
+be written instead.
+
+**Versions**
+
+Please add here the versions of `drift` you're using. It might be relevant to the docs to show somethings version-wise in specific cases.
+
+**Screenshots**
+
+Add screenshots and videos here if needed to help explain your point. For example if something in the website is not behaving correctly.
+
+**Aditional info**
+
+Write down any aditional info/discussion for this issue.
```

Comment:
From experience, most reports don't need versions or screenshots, so it might be a bit annoying for users to remove them from the template every time:

```suggestion
**Aditional info**

Write down any aditional info/discussion for this issue. If you think the documentation issue may be related to your drift version, please include the versions of `drift` you're using.
Also, feel free to add screenshots or videos if they help explaining the documentation issue.
```